src/motor-operations.cc

```diff
@@ -84,6 +85,95 @@ static char test_acceleration_ok(float acceleration) {
 }
 #endif
 
+struct HistorySlot {
+  int steps[BEAGLEG_NUM_MOTORS];
+  int max_axis_steps;
+};
+
+typedef RingDeque<struct HistorySlot, QUEUE_LEN + 1> SlotHistory;
+typedef FixedArray<unsigned,BEAGLEG_NUM_MOTORS> SlotPartial;
+
+// This class is delegated to handle the update of the absolute
```

Comment:
I don't really understand this sentence. 'whenever this operation as to be performed' .. which operation is meant ? Also, it is not clear who is expected to be calling this.